packages/experimental/RadioGroup/src/Radio/Radio.tsx

```diff
@@ -39,14 +41,21 @@ export const Radio = compose<RadioType>({
 
     // now return the handler for finishing render
     return (final: RadioProps) => {
-      const { label, ...mergedProps } = mergeProps(radio.props, final);
+      const { label, subtext, disabled, ...mergedProps } = mergeProps(radio.props, final);
```

Comment:
Curious why disabled is pulled out here?

Reply:
Pulled this out to test if it might be affecting the focus zone of disabled RadioGroups, but it didn't end up mattering if it's applied or pulled out here. Reverted this change since it doesn't apply to this PR / affect it.